specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/mfe.json

```diff
@@ -2930,6 +2942,266 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/marketplaceSubscriptions": {
+      "get": {
+        "tags": [
+          "MarketplaceSubscription"
+        ],
+        "summary": "List containers.",
+        "operationId": "MarketplaceSubscriptions_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v3 [](http://example.com/codeflow?start=69&length=2)

could you please use v5 for all the newer apis ?

Reply:
noted.

we don't have capacity right now and the are running up against a tight deadline, but I will forward this feedback appropriately and hopefully upgrade to the common-types v5 in our next API iteration - is that ok?

Reply:
we have a work item to track this here: https://msdata.visualstudio.com/Vienna/_workitems/edit/2929140